packages/amplication-server/src/main.ts

```diff
@@ -52,7 +54,16 @@ async function bootstrap() {
     app.enableCors();
   }
 
-  await app.listen(process.env.PORT || 3000);
+  const appMode = process.env.APP_MODE || "http";
+  if (appMode === "https") {
+    const httpsServer = https.createServer(httpsOptions, app.getHttpServer());
+    httpsServer.listen(443);
```

Comment:
Users might still need to customise the port and as there is a variable for it `process.env.PORT`, the code should use that value, and if missing, default to 443